java/code/src/com/redhat/rhn/frontend/xmlrpc/chain/ActionChainHandler.java

```diff
@@ -583,6 +584,34 @@ public Integer addScriptRun(User loggedInUser, Integer sid, String chainLabel,
                 loggedInUser, sid, chainLabel, null, uid, gid, timeout, scriptBody);
     }
 
+    /**
+     * Adds an action to apply highstate on the system.
+     *
+     * @param loggedInUser The current user
+     * @param sid          System ID
+     * @param chainLabel   Label of the action chain.
+     * @return True or false in XML-RPC representation (1 or 0 respectively)
+     * @apidoc.doc Adds an action to apply highstate on the system to an Action Chain.
+     * @apidoc.param #session_key()
+     * @apidoc.param #param_desc(" int ", " sid ", " System ID ")
+     * @apidoc.param #param_desc(" string ", " chainLabel ", " Label of the chain ")
+     * @apidoc.returntype #param_desc(" int ", " actionId ", " The id of the action or throw an exception ")
```

Comment:
I don't know if the extra spaces make a difference in the output docs but let's keep it consistent with the rest nevertheless.

```suggestion
     * @apidoc.param #param_desc("int", "sid", "System ID")
     * @apidoc.param #param_desc("string", "chainLabel", "Label of the chain")
     * @apidoc.returntype #param_desc("int", "actionId", "The id of the action or throw an exception")
```

Reply:
@ cbbayburt  Changes have been made.